packages/common/src/groups/HubGroups.ts

```diff
@@ -142,7 +142,7 @@ export async function createHubGroup(
  */
 export async function fetchHubGroup(
   identifier: string,
-  requestOptions: IUserRequestOptions
+  requestOptions: IHubRequestOptions
```

Comment:
is this technically a breaking change? wondering if we need to release it as such?

Reply:
I did check with Dave and we decided this was something small and not widely used and would have been caught by Typescript if the type was incorrect.